src/fides/api/ops/util/data_category.py

```diff
@@ -20,8 +49,14 @@ def generate_fides_data_categories() -> Type[EnumType]:
 
 def _validate_data_category(data_category: str) -> str:
     """Checks that the data category passed in is currently supported."""
-    valid_categories = DataCategory.__members__.keys()
-    if data_category not in valid_categories:
+    from_db = [cat.fides_key for cat in get_fides_data_categories_from_db()]
+    from_taxonomy = [cat.fides_key for cat in get_fides_data_categories_from_taxonomy()]
```

Comment:
Is the main reason you're having to get data categories from the taxonomy our fides ops testing issue?  If what really matters is what is in the customer's database, should we instead try to work towards that?

Reply:
> Is the main reason you're having to get data categories from the taxonomy our fides ops testing issue?

It's because (AFAIK) we can't be certain Fides is running with the data categories from the taxonomy populated in the application DB.

Reply:
To rephrase, I just meant, can't we check what's in the database? I thought the custom taxonomy is loaded on startup.  (Our ops tests delete but that's separate).

Reply:
Hey @pattisdr, I've had another pass at this [here](https://github.com/ethyca/fides/pull/3147/). The main difference is that it attempts to address the issues in the tests where we can't guarantee the taxonomy data categories are in the DB.